src/events.js

```diff
@@ -25,6 +26,38 @@ module.exports = function(Chart) {
 		return filteredEvents;
 	}
 
+	function startHover(element, e, eventHandlers) {
+		var options = element.options;
+		if (!element.hovering) {
+			// fire hover events
+			['mouseenter', 'mouseover'].forEach(function(eventName) {
+				var handlerName = helpers.getEventHandlerName(eventName);
+				var hoverEvent = helpers.createMouseEvent(eventName, e); // recreate the event to match the handler
+				element.hovering = true;
```

Comment:
For `endHover` the lines setting `element.hovering` and `lastHovered` are outside the `forEach`. Is it possible to do the same here for consistency?

Reply:
I think the reason I did it that way (this was back in November) was because that's how the original code was (looking at the diff, it had `element.hovering = true;` inside the `forEach` when starting hover, and then outside the `forEach` for ending the hover)

I can't immediately see why it would be that way, so I'll make the change and test it when I get some time and then get back to you

Reply:
Since there aren't any tests in this library, back in November I was probably just trying to refactor code without messing with the logic more than I needed to